src/styles/main.scss

```diff
@@ -0,0 +1,77 @@
+$day-size: 100px;
+$border-color: #000;
+$backgroundColor: #eee;
+$backgroundColorHover: #ffbfcb;
```

Comment:
Variable names should be written in kebab-case according to the code style rules. Consider renaming `$backgroundColorHover` to `$background-color-hover`.